build.gradle

```diff
@@ -31,7 +31,11 @@ dependencies {
     minecraft "com.mojang:minecraft:${project.minecraft_version}"
     mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
     modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
-    implementation("org.spongepowered:configurate-hocon:4.1.2")
+
+    modApi(include("org.spongepowered:configurate-core:4.1.2"))
```

Comment:
I think we should move the versions to gradle.properties, but I can do that myself before creating a new release

Reply:
Sure, but this PR doesn't change that.